pkg/virt-controller/services/template_test.go

```diff
@@ -1519,7 +1522,7 @@ var _ = Describe("Template", func() {
 				pod, err := svc.RenderLaunchManifest(&vm)
 				Expect(err).ToNot(HaveOccurred())
 
-				Expect(pod.Spec.Affinity).To(BeEquivalentTo(&kubev1.Affinity{PodAffinity: &podAffinity}))
+				Expect(pod.Spec.Affinity.PodAffinity).To(BeEquivalentTo(&podAffinity))
```

Comment:
Can we be sure that `Affinity` will never be `nil`? Thanks

Reply:
Yes we can be sure because we add podAffinity to the vm in the test.